packages/gen-ai/bff/build.openshift.sh

```
@@ -0,0 +1,54 @@
+elif [[ ! "$REPO_URL" =~ ^https:// ]]; then
+    echo "❌ Error: Remote '${REMOTE}' is not using HTTPS for fetch"
+    echo "💡 Current fetch URL: $REPO_URL"
+    echo "💡 Fix with: git remote set-url ${REMOTE} https://github.com/YOUR_USERNAME/odh-dashboard.git"
+    echo "💡 For SSH push only: git remote set-url --push ${REMOTE} git@github.com:YOUR_USERNAME/odh-dashboard.git"
+    exit 1
+fi
+
+# Get current branch name
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+REPO_ROOT=$(git rev-parse --show-toplevel)
+CONTEXT_DIR=$(realpath --relative-to=$REPO_ROOT $(pwd))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)